tests/cupyx_tests/scipy_tests/signal_tests/test_resample.py

```diff
@@ -190,6 +190,22 @@ def test_resample_methods(self, method, ext, padtype, xp, scp):
 
         return results
 
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_resample2d(self, xp, scp):
+        results = []
+        for nx in range(10000, 100000, 10000):
```

Comment:
NIT: Curious if this test is slow, but doesn't seem over the top.
```suggestion
        for nx in range(10_000, 100_000):
```
or add another size, but one is repeated.

Reply:
This test crushed my IDE twice (but I have a pretty old GPU).
What do you mean by "add another size, but one is repeated"? No size is repeated.

Reply:
sorry must have been asleep when looking at it, read it as an explicit list

> This test crushed my IDE twice (but I have a pretty old GPU).

Maybe it is a bit too excessive/large problems, I doubt this very large sizes matter for the fix, so it may be better to just make it smaller.
(Unless you mean my suggestion, which clearly was nonsense, sorry about that.)

Reply:
No problem :)
The current test works fine. I mean your suggestion.